adserver/api/views.py

```diff
@@ -68,7 +68,8 @@ class AdDecisionView(GeoIpMixin, APIView):
         :<json string force_campaign: Limit results to ads from a specific campaign
 
         :>json string id: The advertisement slug of the chosen ad
-        :>json string text: The text of the ad
+        :>json string text: **Deprecated**:The HTML text of only the ad.
```

Comment:
Do we actually want to deprecate this? I don't think we do. I suspect a lot of people (and probably our ad client) will still want to just do something like `$("#div").html(resp.text)`. I just think that having the text be simply a put together version of the body and the headline is ideal.

Reply:
The client is using the `html` in the response -- which I assume is what most people would want if they are just plopping HTML in?

Reply:
I'm happy not to deprecate it for now tho.

Reply:
You're right! Maybe `text` should just be plain text or maybe we should just get rid of it.

Reply:
Yea, I think people are using it currently, but we should definitely remove once we fully migrate.